setup.py

```diff
@@ -9,11 +9,12 @@
       license='MIT',
       packages=find_packages(),
       zip_safe=False,
+      python_requires='>=3.0',
       entry_points={
         'console_scripts': ['genie = genie.__main__:main']},
       install_requires=[
         'pandas>=0.20.0',
         'synapseclient>=1.9',
         'httplib2>=0.11.3',
         'pycrypto>=2.6.1',
-        'PyYAML>=3.11'])
+        'PyYAML>=5.1'])
```

Comment:
why such a big jump?

Reply:
I was just using an old pyyaml version.  There are new specs about how to load yaml file which made me have to use `5.1`